fedmsg/config.py

```diff
@@ -17,15 +17,16 @@
 #
 # Authors:  Ralph Bean <rbean@redhat.com>
 #
-""" :mod:`fedmsg.config` handles loading, processing and validation of
-all configuration.
+"""
+This module handles loading, processing and validation of all configuration.
```

Comment:
Oxford comma [saves capitalists money](http://loweringthebar.net/2017/03/the-oxford-comma-use-it.html).